front/pages/w/[wId]/builder/data-sources/[name]/index.tsx

```diff
@@ -601,14 +600,9 @@ function DatasourceTablesTabView({
 
         <div className="py-8">
           <ContextItem.List>
-            {tables.map((t) => (
+            {tables.map((t, index) => (
               <ContextItem
-                key={tableKey({
-                  workspaceId: owner.sId,
-                  tableId: t.table_id,
-                  dataSourceId: dataSource.name,
-                  dataSourceViewId: undefined,
-                })}
+                key={index}
```

Comment:
it's usually not recommended to use an index as a key, if for some reason the order is changed or if a new table is added in return of useTables, all items will have to be re-rendered  - why removing the previous tableKey ?

Reply:
`tableKey` here does not make sense since this old code path does not use the data source view at all. I agree with you on not using index, but this code path is soon to be dead.